Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11-alpine
+FROM golang:1.11.3-alpine
```

Comment:
I was under the impression that `go1.11` refers to the latest go 1.11.x tag. Actually I just double checked and that is indeed the case. The question being that, if `go1.11` is cached in docker already, it is updated or not. In that case, making the dependency explicit would be an option. Alternatively we could run `git pull golang:1.11` (or whatever version) to force an update to the latest tag.